### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Vova-Stoliar**
[DEMO LINK](https://Vova-Stoliar.github.io/react_sum/)

### CODE REVIEW

File: src/components/Sum.jsx [hunk 1]
```diff
@@ -1 +1,19 @@
...
+import PropTypes from 'prop-types';
+
+export const Sum = ({ a = 0, b = 0 }) => (
+  <p>
+    {'Sum of '}
+    {a}
+    {' and '}
+    {b}
+    {' is '}
+    {a + b}
```
Comment (AndreySerhovetc) [inline src/components/Sum.jsx:12]: You can use this {`Sum of ${a} and ${b} is ${a + b}`}

**Review by AndreySerhovetc — APPROVED** [review]

File: src/components/Sum.jsx (hunk 1 above)
Comment (varseniuk) [reply src/components/Sum.jsx:12]: Yep. It would be more readable

File: src/components/Sum.jsx [hunk 2]
```diff
@@ -1 +1,19 @@
 // Don't forget to add default values an check types of props
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Sum = ({ a = 0, b = 0 }) => (
```
Comment (varseniuk) [inline src/components/Sum.jsx:5]:
```suggestion
export const Sum = ({ a = 0, b = 0 }) => (
```
You could use defaultProps for this instead

**Review by varseniuk — APPROVED** [review]
Good 👍 
But pay attention to my comments

**Pull request closed without merging**